xml/installation-installation-gui_installer.xml

```diff
@@ -7,6 +7,8 @@
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>Installing via the GUI</title>
+ <!--FIXME: GUI installation will be completely redone. Waiting on new
+ Input Model descriptions and instructions to do any edits here. -->
```

Comment:
Does it make sense to keep this comment?

Reply:
No. It needs to go. I'm working on that section right now. Should I do something about it now? Can you get rid of it?

Reply:
never mind. I'm a little apprehensive. But it seems like I should just be able to make these changes and push them to this branch. So I'm gonna do that.
UPDATE: I started to do that and got several conflicts. Doesn't seem like such a hot idea. How should this be handled?

Reply:
fixed. Bite the bullet. Live a little. I think everything worked okay. I'll wait for the judgement.